src/dota2/decoders.ts

```diff
@@ -258,3 +259,12 @@ export function decodeWearable(rawWearable: any) {
   });
   return iWearableList as IWearbleItem[];
 }
+
+export function decodeEvents(rawEvents: Array<any>) {
+  return rawEvents.map((event) => {
+    return { 
+      gameTime: getAttr(event, 'game_time'),
+      eventType: getAttr(event, 'event_type'), 
```

Comment:
This feels very bad to just parse the `event_type` as a string. It can take values such as 'aegis_picked_up', 'bounty_rune_pickup', or 'roshan_killed' (also tips, but I did not happen to grab the string for tips)


Should we make separate events that extend `IEvent`? One for rosh dying, one for bounty rune spawning, etc.? Any ideas?